libbpf-tools/syscount.h

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Anton Protopopov
+#ifndef __SYSCOUNT_H
+#define __SYSCOUNT_H
+
+struct data_t {
+	__u64 count;
+	__u64 total_ns;
+	__u32 key;
```

Comment:
btw, doesn't seem like you are using key field (and you don't need it, right?) 50% memory reduction right there ;)

Reply:
I am using it in user space: I read items into an array of `struct data_t` and sort it by `count` or `total_ns`, but the `key` part should also be sorted to print it. This came from the "batch" version — this way I don't need any extra copy in user space. But now, comparing to syscall overhead this doesn't count, so I can remove this item and use a different data type in user space to store items.